<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainViewer</class>
 <widget class="QMainWindow" name="MainViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>915</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>915</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainViewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="glView" name="glWindow">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>711</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_model_name">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>10</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>MODEL NAME</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="load_file_button">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>80</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LOAD .OBJ FILE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_movement">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>122</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>MOVEMENT</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_translate_x">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>151</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_translate_y">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>184</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_translate_z">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>217</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="translate_button">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>250</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TRANSLATE</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_move_x">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>152</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_move_y">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>183</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_move_z">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>215</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotation">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>292</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>ROTATION</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider_rotation_z">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>385</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotate_y">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>352</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider_rotation_y">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>352</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotate_z">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>385</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotate_x">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>319</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_button">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>420</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ROTATE</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_rotation_x">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>322</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_scaling">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>462</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>SCALING</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider_scale">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>524</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_button">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>562</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SCALE</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_move_x">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>151</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_move_y">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>183</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_move_z">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>215</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_rotation_z">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>383</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_rotation_y">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>351</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_rotation_x">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>319</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_scale">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>491</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>VERTICES</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="count_vertices">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_edges">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>560</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>EDGES</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="count_edges">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>560</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_file">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>40</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>555</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="screenshot_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>555</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gif_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>555</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>GIF</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glView</class>
   <extends>QOpenGLWidget</extends>
   <header>headers/glview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
